packages/agent/src/agents/modes/interactive.ts

```
@@ -118,39 +98,87 @@
+    function getLatestMessageForMessage(
+      messages: BaseMessage[],
+      MessageClass: typeof HumanMessage
+    ): HumanMessage | null {
+      try {
+        for (let i = messages.length - 1; i >= 0; i--) {
+          if (messages[i] instanceof MessageClass) {
+            return messages[i];
+          }
+        }
+        return null;
+      } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)